libbpf-tools/cachestat.c

```diff
@@ -0,0 +1,208 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+// Copyright (c) 2021 Wenbo Zhang
+//
+// Based on cachestat(8) from BCC by Brendan Gregg and Allan McAleavy.
+// 8-Mar-2021   Wenbo Zhang   Created this.
+#include <argp.h>
+#include <signal.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <time.h>
+#include <bpf/libbpf.h>
+#include <bpf/bpf.h>
+#include "cachestat.h"
+#include "cachestat.skel.h"
+#include "trace_helpers.h"
+
+static struct env {
+	time_t interval;
+	int times;
+	bool timestamp;
+	bool verbose;
+} env = {
+	.interval = 1,
+	.times = 99999999,
+};
+
+static volatile bool exiting;
+
+const char *argp_program_version = "cachestat 0.1";
+const char *argp_program_bug_address = "<bpf@vger.kernel.org>";
```

Comment:
we should probably change this to point to https://github.com/iovisor/bcc/tree/master/libbpf-tools (for all tools). I don't think bpf mailing list is right to accept bugs for these tools. It would be great if you can update this for all the other tools as well as a follow-up PR

Reply:
Ok I'll do this